packages/pharos/README.md

```diff
@@ -90,6 +73,10 @@ To use the tokens as CSS variables, import them like so in your styles entrypoin
 /* More global styles */
 ```
 
+## Using Pharos design tokens
+
+Components in Pharos are styled using design tokens, a tech agnostic way to store variables such as typography, color, and spacing so that Pharos can be shared across platforms. You can use these tokens to help style your own components and pages to ensure our brand is properly expressed to users. The token files are located in the `lib/styles/` directory of the package.
```

Comment:
```suggestion
Components in Pharos are styled using design tokens, a tech agnostic way to store design decisions such as typography, color, and spacing so that Pharos can be shared across platforms. You can use these tokens to help style your own components and pages to ensure the brand is properly expressed to users. The token files are located in the `lib/styles/` directory of the package.
```